.github/workflows/linting.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+###
+### Lints all generic and json files in the whole git repository
+###
+
+name: linting
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
Should we also triggered by any branch ?

Reply:
> Keep in mind - you pay for every second spent in GitHub actions.

Not for public repositories ;-)

Reply:
good point :D